presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -107,6 +161,44 @@ public static double ceiling(@SqlType(StandardTypes.DOUBLE) double num)
         return Math.ceil(num);
     }
 
+    private static SqlScalarFunction decimalCeilingFunction(String name)
```

Comment:
annotation driven definition would be nicer here too - but to do that you have to extend annotation based mechanism to bind expressions on literal variables as function extra parameters.